bmwqemu.pm

```diff
@@ -215,11 +215,17 @@ sub log_call {
     else {
         # key/value pairs
         my @result;
+        my $should_stash = 0;
+        my @tmp_array = @_;
+        while (my ($key, $value) = splice(@tmp_array, 0, 2)) {
+            $should_stash = 1 if ($key eq 'secret' && $value == 1);
+        }
         while (my ($key, $value) = splice(@_, 0, 2)) {
             if ($key =~ tr/0-9a-zA-Z_//c) {
                 # only quote if needed
                 $key = pp($key);
             }
+            $value = "*****" if ($key eq 'text' && $should_stash);
```

Comment:
For now I would stay consistent with the previous string "SECRET STRING"

Reply:
I think it is more nice like that. and it implies directly a _secret_ just looking at it. Would you mind to keep it like this?

Reply:
I think using stars or some other placeholder is nicer, indeed. It would make sense to split that change into an additional commit, though.

Reply:
no problem with that.

Reply:
if we want to change it, how about "(masked)" instead?

Reply:
or [masked]